malicious-site-protection/malicious-site-protection-impl/src/main/kotlin/com/duckduckgo/malicioussiteprotection/impl/data/network/MaliciousSiteService.kt

```diff
@@ -78,6 +78,7 @@ data class MatchResponse(
     val url: String,
     val regex: String,
     val hash: String,
+    val category: String,
```

Comment:
Why do we call them `Feed` but `category` here? which one is how we refer to these in the objective?

Reply:
We normally call them feeds, but the network model uses category instead. What I can do is call this property `feed` and add an annotation to parse from `category`